script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt

```diff
@@ -111,57 +114,57 @@ Eh?"
 [sha:e1a47c61368026df31fc6c54388eccba1815c222]{
 -- Page 160, Offset 21743.
 -- ―――そこには。
-―――Over there.
+There I find―――
 }
 [sha:75acabe0395fea1e757c2696898f081d6d91d43e]{
 -- Page 160, Offset 21744.
 -- 通常の部屋にはある筈のない、異様な<封|のり>が施されていた。
-In an ordinary room where it should never have existed, a strange seal is plastered.
+A bizarre seal, the sort you'd never expect to see in an ordinary room.
 }
 [sha:d5a7b63510f8147228cc5d64294718c7ee47083c]{
 -- Page 161, Offset 21745.
 -- 「……<鉄|てつ>……<格子|ごうし>……？　なんで……？」
 "...An iron... grille...? But why...?"
 }
-[sha:66f57ac9d1bf9ac73c7ae8cadf7a2318d028d443]{
+[offset:21746]{
 -- Page 161, Offset 21746.
 -- 信じられない。
-I don't believe this.
+It's enough to make me doubt my own eyes.
 }
 [sha:f21522ef78660254eae442b4401a8a00e4d8b63c]{
 -- Page 161, Offset 21747.
 -- 信じられないが、目の前にあるモノは本物だ。
-I don't want to, but the thing in front of me is the real deal.
+But unbelievable as it is, the thing in front of me now is definitely real.
 }
 [sha:44fe03efbe94d42619e47b605eb1a0214ccd2202]{
 -- Page 161, Offset 21748.
 -- 槙久の部屋の窓は、一面の壁だった。
-The window of Makihisa's room has turned into a single wall.
+The window of Makihisa's room has been completely walled off.
```

Comment:
"the window of" doesn't really read quite right. Maybe "The window to" (my preference thanks to Shiki's first though being that its blocked to prevent entrance from outside) or "The window in"